.organizationmap

```diff
@@ -222,7 +225,9 @@ Intel <contact@intel.com> Yuan Zhou <yuan.zhou@intel.com>
 Intel <contact@intel.com> Zhiqiang Wang <zhiqiang.wang@intel.com>
 inwinSTACK inc. <leo.l@inwinstack.com> Chang-Yi Lee <cy.l@inwinstack.com>
 inwinSTACK inc. <leo.l@inwinstack.com> Albert Tu <albert.t@inwinstack.com>
+inwinSTACK inc. <leo.l@inwinstack.com> MinSheng Lin <minsheng.l@inwinstack.com>
 IO <contact@io.com> Joe Julian <jjulian@io.com>
+iQIYI <contact@iqiyi.com> Chengwei Yang <yangchengwei@qiyi.com>
```

Comment:
Hi Chengwei Yang,


Le 06/09/16 à 03:19, Chengwei Yang a écrit :

> In .organizationmap 
> <https://github.com/ceph/ceph/pull/10955#discussion_r77562380>:
>
> >  IO <contact@io.com> Joe Julian <jjulian@io.com>
> > +iQIYI <contact@iqiyi.com> Chengwei Yang <yangchengwei@qiyi.com>
>
> Please use email address edison@qiyi.com <mailto:edison@qiyi.com>, thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ceph/ceph/pull/10955/files/add97169cb1c49907fe90ea627edae66d76dacbb#r77562380>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGvyZoNt7XSby8nNqlpNdYMbNG2cE0Dcks5qnL-VgaJpZM4JzIv3>.
>
Ok, is this one better ?

https://github.com/ceph/ceph/pull/10955/commits/f358c504ddf5d5fa883aa4cdc999c1e44fa046cb

Cheers,